### ISSUE TITLE
Guidance on when to use 'add', 'create', and 'new'.

### ISSUE THREAD

**Issue by ashifalinadaf** [grommet/hpe-design-system#3496]
Follow up ticket on https://github.com/grommet/hpe-design-system/issues/3477 

Create guidance on when to use 'add', 'create' and 'new'. 

----------------

Considerations/recommendations:
- Labels should be action oriented, therefor labels should be represented by verbs. --> Therefore "create" and "add"; "new" is eliminated because it is an adjective (i.e. descriptor)
- Add should be used when selecting from an existed set of options
- Create should be used when a new object is being created / new instantiation of an object or event.

Scope of deliverable:
- Paragraph or handful of bullets points to be added to Button guidance

**Comment by ashifalinadaf** [comment]
Working on the initial research on the use of the words 'add', 'create', and 'new'.

**Comment by ashifalinadaf** [comment]
Created a guidance for use of the words 'add', 'create', and 'new'.
[Format 1: ](https://www.figma.com/file/c5YWujXkFYYw4LtVCEOBuR/%233496-Guidance-on-when-to-use-'add'%2C-'create'%2C-and-'new'?type=design&node-id=32-4606&mode=design&t=JhGiLPS9D3FMaEIV-11)
[Format 2: ](https://www.figma.com/file/c5YWujXkFYYw4LtVCEOBuR/%233496-Guidance-on-when-to-use-'add'%2C-'create'%2C-and-'new'?type=design&node-id=37-10002&mode=design&t=JhGiLPS9D3FMaEIV-11)

**Comment by halocline** [comment]
> @KennyAtHPE @SeamusLeonardHPE @SOjaHPE @halocline Please have a look and let me know if anything is missing or changes needed.
> 

I have left comments.

> I was wondering if the same thing can be expanded for labeling **Menu** component and its **list of items.**

Yes, I would expect that the same type of verb oriented language would apply to any call-to-action menu options.

**Comment by KennyAtHPE** [comment]
Connected with Asif. Agree with the comments left in the file. I'd expect all primary buttons to be lead with a verb so 'new' doesn't seem appropriate to me. I wonder if the type of button (default, secondary, etc.) should have different language rules.

**Comment by ashifalinadaf** [comment]
Added the guidance to the use of the words 'create' and 'add'. 
TL;DR 
Add- to add something that already exists. 
create - create something that doesn't exist, from the scratch. 
New- omitting the use of 'new' to keep the users away from confusion and using 'create' instead of 'new'.

After collecting the inputs from the GLCS team/ designers/devs,
Following guidelines is proposed to add it to button guidance: 

**Guidance on when to use 'add', 'create', and 'new'.**

**Add: ‘**Add’ should be used where someone can add **something that is already existing.** 
It’s recommended to include the name of the thing that user can create. Combine add with the object where appropriate, 
**Add+Thing to create.** 
Examples: Add user, Add file, or Add member. 

**Create:** 
‘Create’ should be used where the user wants to create something that needs to be **produced from the scratch**, from nothing. 
Create requires users to include details as part of the create process. Create can be used for submit button of a form.  
Example: Create a design, Create a project, or Create volume. 

**For now thinking of not using the word add, however in case we still want to use, the following guidance can be used.** 
New: Use ‘New’ for creating something new, a new object Include the name of the thing that user can create.
Use ‘New’ to initiate the action. 
Examples: New account, new file or New tab.

**Comment by MikeKingdom** [comment]
Looks pretty good. New is pretty rare but it does work well for some local interface types of things like new tab. Add can be a little less clear in those cases since Add seems to work better for adding something existing and create really works better for creating some data object that will persist on the backend system.

**Comment by SOjaHPE** [comment]
@ashifalinadaf this ticket looks to be in great shape
As @halocline  mentioned in the comments, there is a subtle nuance between using 'new' and 'create'. I do think 'new' explicitly communicates that the user is about to create something that doesn't exist. 'create tab' doesn't seem to have the concise label compared to 'new tab'. It isn't a verb but I is still clear and familiar. I'd recommend that we publish this and get it added to the button guidance. Sooner we can get it published the quicker we can gather feedback and adjust if required. Thank you
